notification/docs/IntegrationGuide.md

```diff
@@ -58,6 +58,14 @@ Replace all `XXX` values and execute:
 docker run -e CTP_PROJECT_KEY=XXX -e CTP_CLIENT_ID=XXX -e CTP_CLIENT_SECRET=XXX ctp-adyen-integration-notification:XXX
 ```
 
+### Deployment using AWS Lambda
+
+##### Deploying the Lambda Code
+For deployment to lambda, zip the notification folder and specify `src/lambda.handler` as the entry point for the function
+
+##### Deploying the API Gateway
+In order for Adyen to call the lambda, an API Gateway will need to be created which accepts the notifications from Adyen and invokes the lambda.
```

Comment:
Could you please link here to API Gateway resources?

Reply:
What resources do you mean, just the AWS API Gateway main page? Or something different?

Reply:
Yes AWS doc.

Reply:
@zackpollard Please add the link here and we are good to go with the merge :)

Reply:
Done :)

Reply:
Thanks. It did not have to be a separate point. It would be enough if you would link only the expression `API Gateway`